<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventTriggerInput</class>
 <widget class="QWidget" name="EventTriggerInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#EventTriggerInput{
background:#26282D;
border-radius:4px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>信号函数：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>信号检查函数：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="input_trigger">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="input_check">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btn_remove">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: transparent;
}
QPushButton:hover{
background: #4C4E56;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/res/delete.svg</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../app.qrc"/>
 </resources>
 <connections/>
</ui>
